app/graphql/queries/fetch_user_aggregate_footprint.rb

```diff
@@ -0,0 +1,13 @@
+module Queries
+  class FetchUserAggregateFootprint < Queries::BaseQuery
+    type Types::AggregateFootprintType, null: true
+
+    argument :user_id, Integer, required: true
+    argument :month, String, required: true
+    argument :year, String, required: true
+
+    def resolve(user_id:, month:, year:)
+      FootprintFacade.aggregate_footprint_for_month(user_id, month, year)
```

Comment:
See comment above. I think we should call Footprint.aggregate_footprint_for_month(). Also, the graphQL query should be changed to FetchUserAggregateMonthFootprint, in case we want to aggregate by year as well.